backend/tests/Notifo.Domain.Tests/Integrations/Discord/DiscordTests.cs

```diff
@@ -0,0 +1,155 @@
+﻿// ==========================================================================
+//  Notifo.io
+// ==========================================================================
+//  Copyright (c) Sebastian Stehle
+//  All rights reserved. Licensed under the MIT license.
+// ==========================================================================
+
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Notifo.Domain.Integrations.Discord;
+
+[Trait("Category", "Dependencies")]
+public sealed class DiscordTests
+{
+    private readonly ResolvedIntegration<DiscordIntegration> sut;
+
+    public DiscordTests()
+    {
+        sut = CreateClient();
+    }
+
+    [Fact]
+    public void Should_get_integration()
+    {
+        Assert.NotNull(sut.System);
+    }
+
+    [Fact]
+    public async Task Should_send_simple_message()
+    {
+        var userId = GetUserId();
+        var message = new MessagingMessage
+        {
+            Text = "Test message",
+            Targets = new Dictionary<string, string>() { { DiscordIntegration.DiscordChatId, userId } }
+        };
+
+        var result = await sut.System.SendAsync(sut.Context, message, default);
+
+        Assert.Equal(DeliveryResult.Handled, result);
+    }
+
+    [Fact]
+    public async Task Should_send_full_message()
+    {
+        var userId = GetUserId();
+
+        var message = new MessagingMessage
```

Comment:
They way you add extra lines is a little bit inconsistent here. I like this method, but other methods look different. Especially before "var message" and before "var result"